Python/cp/house_robber.py

```diff
@@ -0,0 +1,26 @@
+class Solution(object):
+    def rob(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        if nums ==[] or len(nums) == 0:
+            return 0
+        elif len(nums) == 1: 
+            return nums[0]
+        
+
+        runningTotal = [-1,-1]
+        runningTotal[0] = nums[0]
+        runningTotal[1] = max(nums[0], nums[1])
+
+        for index in range(2, len(nums)):
+            runningTotal.append(max([nums[index] + runningTotal[index - 2], runningTotal[index - 1]]))
+
+
+        return runningTotal[-1]
+
+if __name__ == '__main__':
+    nums = [1,2,3,1]
+    instance = Solution()
+    print(instance.rob(nums))
```

Comment:
Add complexity of your program and sample input/output in commented form.

Reply:
@atarax665 Please check the [commit](https://github.com/TesseractCoding/NeoAlgo/pull/1315/commits/6c003d24be03e79e3026859b3a62a6091125483f) now.